.github/workflows/assign_reviewers.yml

```diff
@@ -16,26 +16,26 @@ jobs:
   requested-reviewer:
     runs-on: ubuntu-latest
     steps:
-    - name: Assign requested reviewer
-      uses: actions/github-script@v6
-      with:
-        script: |
-          try {
-            if (context.payload.pull_request === undefined) {
-            throw new Error("Can't get pull_request payload. " +
-                            'Check a request reviewer event was triggered.');
+      - name: Assign requested reviewer
```

Comment:
Prettier changed indentation in a bunch of yaml files. Please confirm that it's still file/the indentation is irrelevant.